src/command/create/editor.ts

```diff
@@ -141,6 +146,66 @@ function vscodeEditorInfo(): EditorInfo {
   return editorInfo;
 }
 
+function positronEditorInfo(): EditorInfo {
+  const editorInfo: EditorInfo = {
+    id: "positron",
+    name: "positron",
+    open: (path: string, createResult: CreateResult) => {
+      const artifactPath = createResult.path;
+      const cwd = Deno.statSync(artifactPath).isDirectory
+        ? artifactPath
+        : dirname(artifactPath);
+
+      return async () => {
+        const p = Deno.run({
+          cmd: [path, artifactPath],
+          cwd,
+        });
+        await p.status();
+      };
+    },
+    inEditor: isPositronTerminal(),
+    actions: [],
+  };
+
+  if (Deno.build.os === "windows") {
+    editorInfo.actions.push({
+      action: "which",
+      arg: "Positron.exe",
+    });
+    const pathActions = windowsAppPaths("Positron", "Positron.exe").map(
+      (path) => {
+        return {
+          action: "path",
+          arg: path,
+        } as ScanAction;
+      },
+    );
+    editorInfo.actions.push(...pathActions);
+  } else if (Deno.build.os === "darwin") {
+    editorInfo.actions.push({
+      action: "which",
+      arg: "positron",
+    });
+
+    const pathActions = macosAppPaths(
+      "Positron.app/Contents/Resources/app/bin/positron",
```

Comment:
@cscheid @gordonwoodhull I need confirmation I got the path correctly on Mac; 

You can also test the PR by running 
`quarto create project blog .` inside Positron Terminal. 

I tested on WIndows

Reply:
I checked out the branch and tried `quarto create project blog .` inside the Positron terminal. (MacOS)

It still opened a new VScode window. 

Anything I can do to test further?

Reply:
yes. Can you check the path I used for the application ? 

What does`which positron` returns on your side ?

Reply:
Thanks @gordonwoodhull for the debugging !